readme.md

```diff
@@ -47,6 +47,7 @@ Please raise a [Pull Request](https://github.com/wizztjh/awesome-webvr/pulls) to
 
 ### Demos
 
+- [ArtSocks](https://art-socks.herokuapp.com/demo.html) - An experiment with VR and web sockets by [dziemid](https://twitter.com/dziemid) (Google Cardboard)
```

Comment:
"An experimental VR art gallery with virtual pointer to share where the user is looking using web sockets by dziemid (Google Cardboard)" Sounds good for you?